configuration/packages.yaml

```diff
@@ -78,6 +78,6 @@ packages:
     buildable: False
     version: ['@0.10.0rc0']
     paths:
-      tensorflow+cuda@0.10.0rc0%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu 
+      tensorflow@0.10.0rc0+cuda@7.5.18+cudnn@4.0.cuda7%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu 
```

Comment:
Well, this version of tensorflow has been compiled with gcc4.9.3 cuda7.5.18 cudnn4.0.cuda7. If you don't have this constraint it does not work